service-loadbalancer/template.cfg

```diff
@@ -4,6 +4,10 @@ global
     daemon
     stats socket /tmp/haproxy
 
+    {{ if eq .startSyslog "true" }}# log using a syslog socket
```

Comment:
does having the if/end formatted this way give us better formatting of the final config? please have this if/end consistent with the ones below, eitherway (if this is better change those, or if it doesn't matter make this like the others.)